<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQTMsgBox</class>
 <widget class="QQTDialog" name="QQTMsgBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>223</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLabel" name="lbContent">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="widgetBottom" native="true">
     <widget class="QQTPushButton" name="btnYes">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Yes</string>
      </property>
     </widget>
     <widget class="QQTPushButton" name="btnNo">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>85</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>No</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQTPushButton</class>
   <extends>QPushButton</extends>
   <header>qqtpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QQTDialog</class>
   <extends>QWidget</extends>
   <header>qqtdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
